tests/fixtures/ecma-version/13/class-static-blocks/expressions-arrow-function-static-init-await-binding.src.js

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
I'll try to add the test262 url here. thoughts?

Reply:
Can you explain what you mean by this? I'm not understanding.

Reply:
I was wondering to add a link to https://github.com/tc39/test262/commit/afe217b318df5f197e64b30a8b5a4b391c777359, as it was copied from there.

Reply:
You’d need to add not just a link but the full copyright and license.

Reply:
great, will do!